UsefulTrophies/PlayerPatch.cs

```diff
@@ -70,7 +70,7 @@ class UseItemPatch
 
                 // Consume Item 
                 inventory.RemoveOneItem(item);
-                __instance.m_consumeItemEffects.Create(Player.m_localPlayer.transform.position, Quaternion.identity, null, 1f);
+                __instance.m_consumeItemEffects.Create(Player.m_localPlayer.transform.position, Quaternion.identity, null, 1f, -1);
```

Comment:
This new argument is not really "missing" in the sense that it needs to be provided a value, since it is an optional parameter. I believe the actual issue with the mod was just that it needed to be built with the new managed assemblies from the updated game. I changed nothing at all and built it in VS community 2019 referencing my assemblies from my install and it is working without issue.